### DISCUSSION TITLE
0.15.0

### DISCUSSION THREAD

**Discussion by navarr** [wardenenv/warden#819] [Announcements]
### Big Changes
* Moved from Mailhog to Mailpit ([#779](https://github.com/wardenenv/warden/pull/779) by @bap14)  
  While this seems like a fairly large change, it is smaller than you may give it credit for.  Email should continue to 
  be pushed to the email catcher, which is now located at https://webmail.warden.test/, with a redirect from the old
  mailhog URL.

### Enhancements
* Updated Laravel templates to latest PHP version ([#776](https://github.com/wardenenv/warden/pull/776) by @norgeindian)
* Updated Laravel to use Composer 2 ([#775](https://github.com/wardenenv/warden/pull/775) by @sprankhub)
* Added success message to `env-init` command ([#765](https://github.com/wardenenv/warden/pull/765) by @hardyjohnson)
* Added a warning message to start warden services if they're not running when you try to start an environment ([#766](https://github.com/wardenenv/warden/pull/766) by @hardyjohnson)

### Bug Fixes
* Fixed bash error "value too great for base" during `warden env up` ([#817](https://github.com/wardenenv/warden/pull/817) by @ihor-sviziev)

## New in Images

Images are regularly updated, and these fixes roll out ASAP after they are merged. This is just a summary of what changes have occurred between the last release and the new one.

* Add CakePHP nginx configuration by @bap14 in https://github.com/wardenenv/images/pull/31
* Add PHP-SPX image build by @bap14 in https://github.com/wardenenv/images/pull/17
* PHP extension PECL swoole by @lfluvisotto in https://github.com/wardenenv/images/pull/33
* Enable nginx to serve image filters by @theuargb in https://github.com/wardenenv/images/pull/32
* Add OroCommerce Images by @bap14 in https://github.com/wardenenv/images/pull/19
* Updating dockerfiles and entrypoint files according to latest system requirements of adobe commerce (magento open source) on-premises by @lfluvisotto in https://github.com/wardenenv/images/pull/34
* Updating nginx magento2.conf according to latest version of nginx.conf in Magento 2 Open Source repository by @lfluvisotto in https://github.com/wardenenv/images/pull/37
* Added versions of services:
  * ElasticSearch 8.15
  * MariaDB 11.5, 11.6-rc
  * MySQL 9.0, 9.1
  * Postgres 16.4, 17.0
  * OpenSearch 2.13, 2.14, 2.15, 2.16, 2.17
  * RabbitMQ 4
  * Redis 7.4
  * Varnish 7.6

## New in Docs
* Updated documentation to latest requires for Adobe Commerce by @lfluvisotto in https://github.com/wardenenv/docs/pull/22
* Added Sphinx autobuild by @SamJUK in https://github.com/wardenenv/docs/pull/24

<hr /><em>This discussion was created from the release <a href='https://github.com/wardenenv/warden/releases/tag/0.15.0'>0.15.0</a>.</em>

**Comment by Winfle** [comment]
Is it planed to make mutagen optional for Mac users?

**Reply by navarr** [reply]
It's considered. I forget if there's a pr already open for it. 

The roadmap is viewable on the project board at https://github.com/orgs/wardenenv/projects/1

We only have two maintainers both with full time jobs so it can be a bit of a struggle to move things along. Any help offered is always welcome

**Reply by Winfle** [reply]
@navarr I think, I can help with this one on next week.
Thanks

**Reply by bap14** [reply]
@Winfle This is specifically being addressed here: https://github.com/orgs/wardenenv/projects/1?pane=issue&itemId=48916821&issue=wardenenv%7Cwarden%7C718
